index.html

```diff
@@ -9,11 +9,10 @@
   </head>
   <body>
     <script type="module">
-      import {ReplaySubject} from 'rxjs'
-      const eventBus = new ReplaySubject()
-
       const plugin = document.createElement('orchy-next-plugin')
-      plugin.eventBus = eventBus
+      plugin.orchyProperties = {
+        nextBase: 'http://localhost:3002'
```

Comment:
I recommend reconfiguring this to come from an env var

Reply:
This is a property injectable by the configuration

Reply:
Ah, got it. I should have tried to understand what I'm commenting on. :)